test/_data/dart_imports_go_first/bad.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Also curious why we need an integration test here.

Couldn't we just put the contents of `bad.dart` in `test/rules/dart_imports_go_first.dart`?